File: src_f/pfem_finalize.f
!C
!C***
!C*** PFEM_FINALIZE
!C***
!C
      subroutine PFEM_FINALIZE
      use pfem_util
      implicit REAL*8 (A-H,O-Z)

      call MPI_FINALIZE (errno)
      if (my_rank.eq.0) stop ' * normal termination'

      return
      end

File: src_c/pfem_finalize.c
/***
 *** PFEM_FINALIZE
 ***/
#include <stdio.h>
#include <stdlib.h>
#include "pfem_util.h"
void  PFEM_FINALIZE()
{
     MPI_Finalize ();

     if( my_rank == 0 ){
       fprintf(stdout,"* normal termination\n");
       exit(0);
     }
}
